libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -333,6 +398,25 @@ static DNNReturnType get_output_ov(void *model, const char *input_name, int inpu
     in_frame->width = input_width;
     in_frame->height = input_height;
 
+    if (ctx->options.input_resizable) {
+        status = ie_network_get_input_shapes(ov_model->network, &input_shapes);
+        input_shapes.shapes->shape.dims[2] = input_height;
+        input_shapes.shapes->shape.dims[3] = input_width;
+        status |= ie_network_reshape(ov_model->network, input_shapes);
+        ie_network_input_shapes_free(&input_shapes);
+        if (status != OK) {
+            av_log(ctx, AV_LOG_ERROR, "Failed to reshape input size for %s\n", input_name);
+            return DNN_ERROR;
+        }
+    }
+
+    if (!ov_model->exe_network || !ov_model->infer_request) {
```

Comment:
is ov_model->exe_network enough for the check?

Reply:
if (!ov_model->exe_network)

Reply:
it is enough for now, but will we change the code in the future which may cause that the ov_model->exe_network is initiated but ov_model->infer_request is not?
So i thought it does no harm to check ov_model->infer_request again here. Or am i overthinking?

Reply:
good thinking.  just think it is enough now, we may add infer_request when needed.

Reply:
OK, i got you